tests/helper/classes/zigzag_runner.py

```diff
@@ -7,6 +7,7 @@
 import os
 import re
 import swagger_client
+from tenacity import *
```

Comment:
Nitpick: Can you instead do `import tenacity` and use fully qualified calls? It's just to enable easier discover of where imported code originates from.